ppb_vector/__init__.py

```diff
@@ -6,6 +6,8 @@
 
 __all__ = ('Vector',)
 
+__version__ = "1.0"
```

Comment:
Can we avoid duplicating that information (which is already in `setup.cfg`)?
Otherwise, it needs to be maintained in 2 places and will eventually get out of sync.